client/gutenberg/extensions/markdown/editor.scss

```diff
@@ -139,6 +139,9 @@ $text-editor-font-size: inherit;
 
 .wp-block-jetpack-markdown {
 	.wp-block-jetpack-markdown__editor {
+		font-family: Menlo, Consolas, monaco, monospace;
+		font-size: 14px;
+
```

Comment:
It looks like this is what we currently define in calypso sass for monospace 
```
$monospace: "Courier 10 Pitch", Courier, monospace;
$code: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", $monospace;
```

Should we be following the same fonts from there?

Reply:
This comes from Gutenberg:

https://github.com/WordPress/gutenberg/blob/286317c8e61b1a2922243875fe1f98c868d7859b/assets/stylesheets/_variables.scss#L10

Ideally, we'll be able to use that variable directly from Gutenberg in the future– maybe we should add a comment along those lines.

Using the same fonts as Gutenberg helps to ensure that they'll be available in all environments. I'd like to leave this as is.

Reply:
Let's use the sass variable from top of this file here:

https://github.com/Automattic/wp-calypso/blob/e0e87ef23b476d86b1d770f3cc6b17c12e0e0718/client/gutenberg/extensions/markdown/editor.scss#L5

Reply:
good finding 😅

Reply:
But of course 🤦‍♂️

Reply:
Thanks @simison, addressed in https://github.com/Automattic/wp-calypso/pull/28799/commits/486785ee08d903240ab2dd61d6819b25ef96db51

@enejb we've intentionally used the styles from Gutenberg, so they're consistent with the code block and the custom HTML blocks there. Eventually, we might want to align Calypso with this as well.